### PR TITLE
Update documentation

### LINKED ISSUE #1114: Didn't receive notification about security vulnerability in repository
Are you experiencing this issue in octobox.io or your own instance?
 - [x] octobox.io
 - [ ] My instance

🐞 **Problem**

I recently received an email from GitHub about a security vulnerability on a repository that I am an admin on (and also an owner of the organisation that the repository lives in), however, I didn't see an equivalent notification in Octobox.

@andrew mentioned on Twitter that this doesn't always work:

https://twitter.com/teabass/status/1052461092864028672

And requested the creation of an issue to track it.

🎯 **Goal**

My personal goal is to completely remove the dependency on email, and have all my GitHub interactions happen via Octobox as it is simply amazing! 😄 

💡 **Possible solutions**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by BenJam**
closes #1114

WIP

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,9 +80,15 @@ You can also install the cross-browser [Octobox Web Extension](https://github.co
 
 ## Requirements
 
-[Web notifications](https://github.com/settings/notifications) must be enabled in your GitHub settings for Octobox to work.
+[Web notifications](https://github.com/settings/notifications) must be enabled in your GitHub settings for Octobox to work. To receive vulnerability notifications you must enable these also. 
 
-<img width="757" alt="Notifications settings screen" src="https://cloud.githubusercontent.com/assets/1060/21509954/3a01794c-cc86-11e6-9bbc-9b33b55f85d1.png">
+<img width="739" alt="screenshot 2018-11-12 at 14 32 38" src="https://user-images.githubusercontent.com/158833/48353900-6046e600-e688-11e8-9325-eaaa30f4190e.png">
```
Comment (andrew) [inline README.md]: Can we put this image in the repo so it works offline too?

File: README.md [hunk 2]
```diff
@@ -80,9 +80,15 @@ You can also install the cross-browser [Octobox Web Extension](https://github.co
 
 ## Requirements
 
-[Web notifications](https://github.com/settings/notifications) must be enabled in your GitHub settings for Octobox to work.
+[Web notifications](https://github.com/settings/notifications) must be enabled in your GitHub settings for Octobox to work. To receive vulnerability notifications you must enable these also.
```
Comment (pradyunsg) [inline README.md]:
```suggestion
[Web notifications](https://github.com/settings/notifications) must be enabled in your GitHub settings for Octobox to work. To receive vulnerability notifications, you must enable them as well.
```

File: README.md (hunk 1 above)
Comment (BenJam) [reply README.md]: not sure whether #c918988 is what's intended 💁‍♂️

File: README.md [hunk 3]
```diff
@@ -82,7 +82,7 @@ You can also install the cross-browser [Octobox Web Extension](https://github.co
 
 [Web notifications](https://github.com/settings/notifications) must be enabled in your GitHub settings for Octobox to work. To receive vulnerability notifications, you must enable them as well.
 
-<img width="739" alt="screenshot 2018-11-12 at 14 32 38" src="https://user-images.githubusercontent.com/158833/48353900-6046e600-e688-11e8-9325-eaaa30f4190e.png">
+<img width="739" alt="screenshot 2018-11-12 at 14 32 38" src="/docs/assets/48353900-6046e600-e688-11e8-9325-eaaa30f4190e.png">
```
Comment (andrew) [inline README.md]: Can we use a more descriptive filename? 😉

**Pull request merged**